tapMainExperiment.m

```diff
@@ -73,12 +73,6 @@
         % % % do we need cell array or character would suffice? 
         currSeq = makeSequence(cfg,seqi);
 
-        % fill the buffer
-        PsychPortAudio('FillBuffer', cfg.pahandle, [currSeq.outAudio;currSeq.outAudio]);
-
-        % start playing
-        currSeqStartTime = PsychPortAudio('Start', cfg.pahandle, cfg.PTBrepet,...
-            cfg.PTBstartCue, cfg.PTBwaitForDevice);
```

Comment:
I sort of understood why but wanna be sure. Why do you record the stimuli into logfile first, and while playing the sounds recording the tapping?